__tests__/app.test.js

```diff
@@ -67,13 +67,46 @@ describe("/api/articles", () => {
       });
   });
 });
+
+describe("get article by id", () => {
+  it("200: resolves with article", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        expect(article).toEqual([
+          {
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+          },
+        ]);
+      });
+  });
+
+  it("404: returns message if it cannot find article with that id", () => {
+    return request(app)
```

Comment:
this is great! There is one more test you need here and it's for a 400 error - what if a user tries to put a string for the article_id ?